editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search/facets.ts

```diff
@@ -0,0 +1,29 @@
+import { IndexedNode } from "@opensystemslab/planx-core/types";
+import { FuseOptionKey } from "fuse.js";
+
+type SearchFacets = Array<FuseOptionKey<IndexedNode>>;
```

Comment:
It isn't really obvious to me what 'facets' are but I see that they are already a named thing in the codebase and possibly a standard word in search components! 😅

Reply:
It was also a new term to me! https://en.wikipedia.org/wiki/Faceted_search